src/components/util/deploy.tsx

```diff
@@ -11,7 +11,11 @@ export const deployDocumentStore = async (
     const signer = await getSigner();
     log ? log("Wallet successfully decrypted.") : null;
     const factory = new DocumentStoreFactory(signer);
-    log ? log(`Deploying document store "${storeName}".`) : null;
+    log
```

Comment:
I think I already said we should add the link to the TX, didn't I?

Reply:
Okay, I have added the link to TX. in addition I minor changes in the message to reflect the changes at the top.